dbt/adapters/presto/connections.py

```diff
@@ -24,6 +24,8 @@ class PrestoCredentials(Credentials):
     user: str
     password: Optional[str] = None
     method: Optional[str] = None
+    http_headers: Optional[Dict[str, str]] = None
+    http_scheme: Optional[str] = None
```

Comment:
Would you mind adding these to the README ([table of profile options](https://github.com/dbt-labs/dbt-presto/blob/master/README.md#configuring-your-profile))?